src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/category/CategoriesFieldset.java

```diff
@@ -95,13 +127,15 @@ public CategoriesFieldset clickEdit() {
 
   /**
    * For mobile "All" category is not visible, that is why it's always false - can not be edited.
-   *
+   * TODO: please remember to change this line when SOC-3793 is done:
+   * TODO: "isCategoryEditable(editableCategoryAll)"
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1135)